pkg/injector/envoy_config_test.go

```diff
@@ -29,7 +29,6 @@ var _ = Describe("Test functions creating Envoy bootstrap configuration", func()
 	const (
 		containerName = "-container-name-"
 		envoyImage    = "-envoy-image-"
-		clusterID     = "-cluster-id-"
```

Comment:
We need to keep this here - this is used in the Envoy config (it was here before Multicluster)

Reply:
I'm looking for what breaks the CI. Thanks.